pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[tool.ruff]
+target-version = "py37"
+line-length = 88
+select = [
+    "E", # pycodestyle errors
+    "EM", # flake8-errmsg
+    "F", # pyflakes errors
+    "I", # isort
+    "ISC", # flake8-implicit-str-concat
+    "PGH", # pygrep-hooks
+    "RUF100",  # unused noqa (yesqa)
+    "S", # flake8-bandit
+    "W", # pycodestyle warnings
+    "YTT", # flake8-2020
+]
+ignore = [
+    # TODO: these could be enabled later
+    "S101", # use of assert
+    "S110", # try-except-pass
+    "S112", # try-except-continue
+]
+
+[tool.ruff.isort]
+known-first-party = ["PIL"]
+
+[tool.ruff.per-file-ignores]
+# Ignore Bandit in tests
+"Tests/*" = ["S"]
+"setup.py" = ["S"]
+# TODO: these files have some ambiguous `l` identifiers,
+#       which should be cleaned up.
+"src/PIL/BdfFontFile.py" = ["E741"]
+"src/PIL/Image.py" = ["E741"]
+"src/PIL/ImageFile.py" = ["E741"]
+"src/PIL/PcfFontFile.py" = ["E741"]
+"src/PIL/TiffImagePlugin.py" = ["E741"]
```

Comment:
@hugovk Instead of diving deep into various font formats etc. right now, I opted to disable these codes for those files for now; I can gladly add subsequent PRs, but those would definitely be easier to review separately.

Reply:
Sure. Maybe the per-line `noqa`s are better than per-file ignores, to prevent new ones being introduced?

Reply:
@hugovk Well, I was planning on cleaning these up in subsequent PRs in the following days – and should new ones get introduced in that timeframe, it shouldn't be a big problem to clean them up then.